packages/terra-date-picker/tests/nightwatch/date-picker-spec.js

```diff
@@ -68,5 +67,18 @@ module.exports = {
     browser.expect.element('.terra-DatePicker-range > .terra-DatePicker').to.be.present;
   },
 
+  'Displays the calendar button with a height that matches the input ': (browser) => {
+    browser.url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/date-picker-tests/default`);
+
+    let inputHeight;
+
+    browser.getElementSize('.terra-DatePicker-input', function getHeight(result) {
```

Comment:
What I'm testing here is using [getElementSize](http://nightwatchjs.org/api#getElementSize) to get the height of the input and calendar button and expect that height values match. Once I have both height values, the way I found to compare two standalone values in nightwatch is using this.asset.equal() inside getElementSize. However, if I changed to use arrow-callback, the `this` keyword [no longer binds](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#No_binding_of_this) to the current scope and I can’t use assert.equal(). It is binded to the outer scope. This is the reason I couldn’t use arrow callback.

I couldn’t use getElementSize to get the height and then use assert.cssProperty or expect.css to compare the value because they return different height values for the same element. I think because of rounding, getElementSize returns 25 where as cssProperty returns 25.984375px.

After digging for more night watch APIs, there is actually getCssProperty that I can use to get the height, which returns a consistent value. In using getCssProperty, I no longer need to use this.assert.equal() to compare the values.
https://github.com/cerner/terra-core/pull/481/commits/f55037147df9d3f2b72601095c296500b22e71a6